<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WndSpy</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="1" column="0" rowspan="3" colspan="2">
     <widget class="QTabWidget" name="tabWidget_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="2" column="1">
          <widget class="QCheckBox" name="chkHiddenWindows">
           <property name="text">
            <string>Hidden windows</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="chkMinimize">
           <property name="text">
            <string>Minimize</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="3">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2" rowspan="4">
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionFileExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <widget class="QMenu" name="menuLanguages">
     <property name="title">
      <string>&amp;Languages</string>
     </property>
     <property name="icon">
      <iconset resource="../res/resource.qrc">
       <normaloff>:/images/options-languages.png</normaloff>:/images/options-languages.png</iconset>
     </property>
    </widget>
    <addaction name="menuLanguages"/>
    <addaction name="separator"/>
    <addaction name="actionOptionsTopMost"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Current window</string>
    </property>
    <widget class="QMenu" name="menuCapturar">
     <property name="title">
      <string>&amp;Capture</string>
     </property>
     <addaction name="actionCaptureClient"/>
     <addaction name="actionCaptureWindow"/>
    </widget>
    <addaction name="actionWindowRefresh"/>
    <addaction name="separator"/>
    <addaction name="actionWindowFlash"/>
    <addaction name="menuCapturar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelpAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFileExit">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/file-exit.png</normaloff>:/images/file-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionOptionsTopMost">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Always visible</string>
   </property>
  </action>
  <action name="actionWindowRefresh">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/action-refresh.png</normaloff>:/images/action-refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionWindowFlash">
   <property name="text">
    <string>Flash</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionCaptureClient">
   <property name="text">
    <string>Client area</string>
   </property>
  </action>
  <action name="actionCaptureWindow">
   <property name="text">
    <string>Window area</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/help-about.png</normaloff>:/images/help-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionFileExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWindowRefresh</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotRefresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWindowFlash</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotFlash()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCaptureClient</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotCaptureClient()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCaptureWindow</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotCaptureWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelpAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOptionsTopMost</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>slotTopMost(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotExit()</slot>
  <slot>slotRefresh()</slot>
  <slot>slotFlash()</slot>
  <slot>slotCaptureClient()</slot>
  <slot>slotCaptureWindow()</slot>
  <slot>slotAbout()</slot>
  <slot>slotTopMost(bool)</slot>
 </slots>
</ui>
